src/lsptoolshost/languageStatusBar.ts

```diff
@@ -19,6 +19,7 @@ export function registerLanguageStatusItems(
     if (!getCSharpDevKit()) {
         WorkspaceStatus.createStatusItem(context, languageServerEvents);
     }
+    ProjectContextStatus.createStatusItem(context, languageServer, languageServerEvents);
```

Comment:
Check that this works with devkit - not sure what happens when there are two extensions both contributing items

Reply:
<img width="370" alt="image" src="https://github.com/user-attachments/assets/20e403e3-6aaf-4564-af12-360378cad7f1">